addons/web/static/src/webclient/settings_form_view/widgets/res_config_dev_tool.js

```diff
@@ -33,6 +33,13 @@ class ResConfigDevTool extends Component {
     onClickForceDemo() {
         this.action.doAction("base.demo_force_install_action");
     }
+
+    /**
+     * Downloads every XSD file, based on installed localisations.
+     */
+    onClickDownloadXSD() {
+        this.action.doAction("account.action_download_xsd");
+    }
```

Comment:
This code should not be in the `web` addon but you should extend this component in `account`, as you already did with the template (make it a primary extension instead of altering the original template).
You could then replace the component in the `view_widgets` registry.